### PR TITLE
Feature/page structure

### PR DESCRIPTION
**Pull request by Bluesmile82**
Update CMS structure:

- Page: Different sections of the app
  + default_basemap
  + title
  + slug
  + layer_slugs (many)
  + color: color in the menu (not used yet)
  + external_url (only for scientific_evidence)
  
- Layer Group: 
  + title
  + layers (many)
 
Use a custom map-style to display the basemaps and labels. The basemaps and layers have metadata groups that help switching them on and off. This is the same functionality as in the cms scaffold.

Currently we are using part of the the https://github.com/maputnik/osm-liberty

- Also, added attributions for the map

Please ask me for an import data to test the features if needed

### CODE REVIEW

File: client/src/containers/section/map/attribution.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import { mapSettingsAtom } from '@/store/index';
+
+import { useTheme } from '@/hooks/ui/theme';
+
+const Attribution = () => {
+  const { basemap } = useRecoilValue(mapSettingsAtom);
+  const theme = useTheme('text');
+  const attributionLinkText = {
+    'basemap-satellite': { text: 'ESRI', href: 'https://www.esri.com/home' },
+    'basemap-light': { text: 'Carto', href: 'https://carto.com/' },
```
Comment (clementprdhomme) [inline client/src/containers/section/map/attribution.tsx:12]: Those are not the correct attributions. Have a look at [ORC-81](https://vizzuality.atlassian.net/browse/ORC-81).

Also, please note that OSM Liberty needs further [attributions](https://github.com/maputnik/osm-liberty/blob/gh-pages/LICENSE.md#design-license):
>[© OpenMapTiles](https://openmaptiles.org/) [© OpenStreetMap contributors](https://www.openstreetmap.org/copyright)

File: client/src/containers/section/map/index.tsx [hunk 2]
```diff
@@ -144,7 +147,8 @@ export default function MapContainer() {
         bounds={tmpBounds}
         minZoom={minZoom}
         maxZoom={maxZoom}
-        mapStyle={'https://demotiles.maplibre.org/style.json'}
+        // @ts-expect-error TODO: Map style is not matching the type
+        mapStyle={mapStyle}
```
Comment (clementprdhomme) [inline client/src/containers/section/map/index.tsx:151]: You can cast the variable:
```suggestion
        mapStyle={mapStyle as MapboxStyle}
```

`MapboxStyle` comes from `react-map-gl`.

File: client/src/hooks/pages copy/index.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { useGetPagesId } from '@/types/generated/page';
```
Comment (clementprdhomme) [inline client/src/hooks/pages copy/index.ts:1]: Is the path of this file correct?

Comment (Bluesmile82) [reply client/src/hooks/pages copy/index.ts:1]: It is, why?

File: client/src/containers/section/map/attribution.tsx (hunk 1 above)
Comment (Bluesmile82) [reply client/src/containers/section/map/attribution.tsx:12]: Do we need the Powered by?

File: client/src/hooks/pages copy/index.ts (hunk 3 above)
Comment (clementprdhomme) [reply client/src/hooks/pages copy/index.ts:1]: The path of the file is `pages copy/index.ts`. It seems like “copy” is a mistake, no?

**Pull request closed without merging**